python/python/glide/async_commands/core.py

```diff
@@ -2293,6 +2293,39 @@ async def zpopmax(
             ),
         )
 
+    async def bzpopmax(
+        self, keys: List[str], timeout: float
+    ) -> Optional[List[Union[str, float]]]:
+        """
+        Blocks the connection until it removes and returns a member with the highest score from the first non-empty
+        sorted set, with the given keys being checked in the order they are provided.
+
+        When in cluster mode, all keys must map to the same hash slot.
+
+        BZPOPMAX is the blocking variant of ZPOPMAX.
+
+        BZPOPMAX is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
```

Comment:
```suggestion
        `BZPOPMAX` is the blocking variant of `ZPOPMAX`.

        `BZPOPMAX` is a client blocking command, see https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands for more details and best practices.
```

Reply:
maybe we should do:

` BBZPOPMAX is a client blocking command, see [Blocking commands](https://github.com/aws/glide-for-redis/wiki/General-Concepts#blocking-commands) for more details and best practices. `
 
 
![image](https://github.com/aws/glide-for-redis/assets/116083498/ce221373-3e6b-4376-b9b4-2d34fbdfc39b)

Reply:
I'm not sure if hyperlinks like these are supported by default, according to [this stack overflow post](https://stackoverflow.com/a/73081489/22506474) they aren't, but some tools and/or documentation generators may support them. Regardless, I'll change this here, let me know if you want me to change it back